pkg/instrumentation/sdk_test.go

```diff
@@ -259,7 +529,7 @@ func TestSDKInjection(t *testing.T) {
 							Env: []corev1.EnvVar{
 								{
 									Name:  "OTEL_SERVICE_NAME",
-									Value: "explicitly_set",
+									Value: "explicit-name",
```

Comment:
OOC – why change this for the test?

Reply:
to make sure it's coming from the right place